api/controllers/musicians/musicianGroup.ts

```diff
@@ -7,6 +7,48 @@ import type { getHTTPCode, getPathParams, getResponsesBody } from '@typing';
 import type { NextFunction } from 'express';
 
 type GetMusicianGroupMembership = operations['getMusicianGroupMembership'];
+type GetMusicianGroups = operations['getMusicianGroups'];
+
+export const getMusicianGroups = async (
+  req: Request<
+    getPathParams<GetMusicianGroups>,
+    getResponsesBody<GetMusicianGroups>,
+    {},
+    {}
+  >,
+  res: core.Response<getResponsesBody<GetMusicianGroups>, {}>,
+  next: NextFunction,
+): Promise<core.Response<getResponsesBody<GetMusicianGroups>, {}>> => {
+  try {
+    const musicianGroupRepository = getRepository(MusicianGroup);
+
+    const musicianGroup = await musicianGroupRepository.find({
+      where: {
+        musician: {
+          id: req.params.musicianId,
+        },
+      },
+      relations: [
+        'musician',
+        'group.members',
+        'group.members.musician',
+        'group.genres',
+        'group.events',
```

Comment:
Why do you also need the `members` and the `events` of the group ? I think it would be better to return just the `groupDescription` and if the user want more information, you can fetch the `GET /groups/{groupId}`. WDYT ? 

Here is the `groupDescription` schema : 

```ts
groupDescription: {
      type: 'object',
      required: ['name', 'description', 'location', 'genres'],
      properties: {
        name: { type: 'string' },
        description: { type: 'string' },
        location: { type: 'string', enum: ['Douai', 'Lille'] },
        genres: {
          type: 'array',
          items: { $ref: '#/components/schemas/genre' },
        },
      },
    }
```